src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -319,11 +377,9 @@
               "items": {
                 "type": "string"
               }
-            },
-            "nullable": true
```

Comment:
`HttpValidationProblemDetails` can't actually be `null` when returned via the response.